src/App.js

```diff
@@ -84,6 +84,30 @@ function App() {
     setEventList(newEventList);
     setCustomEvents((currentList) => [...currentList, newEvent]);
   };
+  const removeEvent = (eventName, startTime, endTime) => {
+    const jsonObj = JSON.parse(localStorage.getItem(customKey)) || [];
+
+    const newCustomList = [];
+    jsonObj.forEach((delEvent) => {
+      console.log(startTime, delEvent.title);
+
+      if (
+        delEvent.title === eventName
+        && delEvent.start === startTime
+        && delEvent.end === endTime
+      ) {
+        return;
+      }
+      newCustomList.push(delEvent);
+    });
```

Comment:
This _could_ be replaced by a filter method, but it probably won't impact performance anyways